src/contracts/libraries/GPv2Order.sol

```diff
@@ -139,7 +139,7 @@ library GPv2Order {
         bytes32 structHash;
 
         // NOTE: Compute the EIP-712 order struct hash in place. As suggested
-        // in the EIP proposal, noting that the order struct has 10 fields, and
+        // in the EIP proposal, noting that the order struct has 12 fields, and
         // including the type hash `(12 + 1) * 32 = 416` bytes to hash.
```

Comment:
```suggestion
        // prefixing the type hash `(1 + 12) * 32 = 416` bytes to hash.
```

Reply:
Maybe we can adjust this line as well to make it clearer that the `TYPE_HASH` comes first? AFAIU, this was the other point of confusion.

Reply:
Feel free to not include this suggestion if you don't think its needed.

Reply:
Nice idea!